iwyu_tool.py

```diff
@@ -153,9 +154,22 @@ def main(compilation_db_path, source_files, verbose, formatter, iwyu_args):
 
     # Run analysis
     try:
+
+        pool = Pool()
+        # results doesn't contain actual results, it serves exception handeling
```

Comment:
This comment could be clearer. Perhaps:

    # No actual results in `results`, it's only used for exception handling.
    # Details here: https://stackoverflow.com/a/28660669.